tests/create_private_zone_association/main.tf

```diff
@@ -0,0 +1,21 @@
+provider "aws" {
+  alias   = "vpc_owner"
+  profile = "awsalternate"
+}
+
+resource "aws_vpc" "this" {
+  cidr_block = "10.10.10.0/24"
+  provider   = aws.vpc_owner
+}
+
+module "vpc_association" {
+  source = "../..//modules/vpc_association"
+  providers = {
+    aws           = aws
+    aws.vpc_owner = aws.vpc_owner
+  }
+
+  vpc_id  = aws_vpc.this.id
+  zone_id = "Z036807920UNP3D50TI9S"
```

Comment:
The test can't rely on a hard-coded value like this, since it is dependent on a specific environment. The test _could_ create a zone, but then it would be essentially the same as `create_private_all`. So I think it is fine to just remove this test directory also.